metainfo.patch

```diff
@@ -1,12 +1,13 @@
 diff --git a/dists/org.scummvm.scummvm.metainfo.xml b/dists/org.scummvm.scummvm.metainfo.xml
-index bf086b44338..f288f0f0a59 100644
+index bf086b44338..9c50c12afd3 100644
 --- a/dists/org.scummvm.scummvm.metainfo.xml
 +++ b/dists/org.scummvm.scummvm.metainfo.xml
-@@ -332,4 +332,19 @@ Blade Runner та незліченної кількості інших, ви з
+@@ -332,4 +332,20 @@ Blade Runner та незліченної кількості інших, ви з
      <content_attribute id="money-purchasing">none</content_attribute>
      <content_attribute id="money-gambling">none</content_attribute>
    </content_rating>
 +  <releases>
++    <release version="2.9.0" date="2024-12-12"/>
```

Comment:
The release date is just a placeholder and will be changed to an actual one before merging this PR.